classes/Event/Entity.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Neat\Object\Event;
+
+trait Entity
+{
+    /** @var object */
+    private $entity;
+
+    /**
+     * Event constructor
+     *
+     * @param object $entity
+     */
+    public function __construct($entity)
+    {
+        $this->entity = $entity;
+    }
+
+    /**
+     * @return object
+     */
+    public function entity()
+    {
+        return $this->entity;
+    }
```

Comment:
Is it really necessary to keep this in a separate trait?

Reply:
Not really, not for now. It would've been useful for Events that aren't specific to one Entity. Since we have none, it's indeed better to merge it with the base class.